modules/gapi/samples/pipeline_modeling_tool/dummy_source.hpp

```diff
@@ -18,6 +18,7 @@ class DummySource final: public cv::gapi::wip::IStreamSource {
                 const bool         drop_frames);
     bool pull(cv::gapi::wip::Data& data) override;
     cv::GMetaArg descr_of() const override;
+    double getLatency() const { return m_latency; };
```

Comment:
other methods don't have `get` in their name. Just for consistency

Reply:
Done